Cargo.toml

```diff
@@ -81,6 +81,16 @@ name = "centralized"
 required-features = ["demo", "libp2p/rsa"]
 path = "examples/centralized/validator.rs"
 
+[[example]]
+name = "centralized-web-server"
+required-features = ["full-ci"]
```

Comment:
Would it make sense for both of these to be `"demo", "libp2p/rsa"`? I think they're both executor agnostic.